build/dyn_shim_detection.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+import os
+import subprocess
+import xml.etree.ElementTree as ET
+
+_log = logging.getLogger("dyn-shim-detection")
+show_version_info = project.getProperty("dyn.shim.trace")
```

Comment:
It'd be nice to have a comment explaining where `project` is coming from, and a block comment in general explaining what this script is doing

Reply:
Thanks for reviewing. I have added a comment explaining, I hope it helps.